go/vt/vtgate/vindexes/vschema.go

```diff
@@ -245,6 +248,99 @@ func buildKeyspaces(source *vschemapb.SrvVSchema, vschema *VSchema) {
 		vschema.Keyspaces[ksname] = ksvschema
 		ksvschema.Error = buildTables(ks, vschema, ksvschema)
 	}
+
+	// In the second loop, build references.
```

Comment:
Could probably do all of this in one loop but code would be kinda complicated and don't think this is so a little inefficiency shouldn't hurt I think.